src/integTest/scala/kamon/jmx/collector/KamonJmxCollectorSpec.scala

```diff
@@ -49,31 +48,31 @@ class KamonJmxCollectorSpec extends FlatSpec with Eventually {
 
     implicit val system: ActorSystem = ActorSystem()
 
-    Kamon.addReporter(new PrometheusReporter())
+    Kamon.init()
```

Comment:
The integration test used to query the HTTP port exposed by the `PrometheusReporter`; given you removed it, how does it assert the metrics then?

Reply:
My understanding is that `init` registers the `PrometheusReporter` because it is in the classpath...